CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 
 [Commit Log](https://github.com/xh/hoist-react/compare/v30.0.0...develop)
 
+### ⚙️ Technical
+
+* `Select` will now default to selecting contents on focus if in filter or creatable mode.
```

Comment:
This needs to get shifted up before the commit log link. Agree I think Technical is the correct category for it and that it's worth a one-liner in the changelog.

Reply:
done